test/Main.hs

```diff
@@ -26,9 +26,9 @@ backendTests :: T.TestTree
 backendTests =
   T.testGroup
     "Backend tests"
-    [ ArithmeticCircuit.backendCircuit,
-      LLVM.backendLLVM,
-      Michelson.backendMichelson
+    [ ArithmeticCircuit.backendCircuit
+    -- LLVM.backendLLVM,
```

Comment:
Why are these commented now?

Reply:
Omg, this wasn't supposed to be committed D: